client/me/purchases/credit-cards/index.jsx

```diff
@@ -72,7 +72,7 @@ class CreditCards extends Component {
 		return (
 			<div className="credit-cards">
 				<QueryStoredCards />
-				<SectionHeader label={ this.props.translate( 'Manage Your Credit Cards' ) }>
+				<SectionHeader label={ this.props.translate( 'Manage your credit cards' ) }>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.